### PR TITLE
Add documentation about bot Mattermost plugins

### PR DESCRIPTION
**Pull request by moreauxh**

### CODE REVIEW

File: languages/en/user-guide/bot-mattermost-plugins.rst [hunk 1]
```diff
@@ -0,0 +1,45 @@
+.. _botmattermost-plugins:
+
+Bot Mattermost plugins
+======================
+
+Configuration
+-------------
+
+.. NOTE:: These plugins must be available by your site admin. To administrate these plugins, go to the :ref:`botmattermost-plugins-admin` section.
```
Comment (LeSuisse) [inline languages/en/user-guide/bot-mattermost-plugins.rst]: Should probably be something like "must be made available".

File: languages/en/user-guide/bot-mattermost-plugins.rst [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+To use this feature, **BotMattermost-Agiledashboard plugin** need to be installed and available for the project you need to use.
+
+Go in your project and select **Agile Dashboard** service. In **Admin** section click on **Notification** pane.
+
+.. image:: ../images/screenshots/bot-mattermost-plugins/botADConf.png
+    :alt: Bot agiledashboard configuration
+    :align: center
+
+Fill the form and select the bot charge to send stand-up summary.
+Now, every day, you will receive stand-up summary in your **Mattermost** after the time filled.
```
Comment (LeSuisse) [inline languages/en/user-guide/bot-mattermost-plugins.rst]: What do think about "select the bot that should send the stand-up summary"?

Comment (LeSuisse) [inline languages/en/user-guide/bot-mattermost-plugins.rst]: What do you think about "you will receive your stand-up summary in your **Mattermost** every day after the report time"?

File: languages/en/user-guide/bot-mattermost-plugins.rst [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+.. _botmattermost-git:
+
+BotMattermost-Git
+`````````````````
+
+To use this feature, **BotMattermost-Git plugin** need to be installed and available for the project you need to use.
+
+Go in your tuleap-git repository and click on **Settings**. In the **Notification** section,
+you have the possibility to link a bot previously created at the repository.
```
Comment (LeSuisse) [inline languages/en/user-guide/bot-mattermost-plugins.rst]: s/at/to/

File: languages/en/user-guide/bot-mattermost-plugins.rst (hunk 2 above)
Comment (moreauxh) [inline languages/en/user-guide/bot-mattermost-plugins.rst]: perfect

Comment (moreauxh) [inline languages/en/user-guide/bot-mattermost-plugins.rst]: Yes, it's better.

**Pull request merged**